### ISSUE TITLE
Latest Legendary versions receive virus warning

### ISSUE THREAD

**Issue by Beats24-7** [derrod/legendary#215]
Hey there,
the latest versions of your exe triggers my antivirus software and blocks the execution. What did you change? The latest version that is working without any issues is 0.20.1.

Thanks

**Comment by derrod** [comment]
Duplicate of #207 and #159.

It's a false positive because antivirus software is shit.

**Comment by CommandMC** [comment]
Legendary is not malicious software. False positives exist and happen (one of the "bigger" Antivirus that detects Legendary is AVG). For your own safety, please only use Windows Defender. 3rd party Antiviruses are only made to make money

**Comment by Beats24-7** [comment]
Yes, I know that. I was just curious why it didn't trigger before and now it did. Just using a free scanner.

I updated to the latest version now but my games list still didn't get updated and one game is missing. Is there a minium amount of time for the list to get updated? Added the game a few hours ago but it still doesn't show up (tested 2 different accounts). Keep up the good work.

Thanks again
